packit/api.py

```diff
@@ -347,6 +354,15 @@ def _handle_sources(self, add_new_sources, force_new_sources):
                 archive = self.dg.download_upstream_archive()
                 self.dg.upload_to_lookaside_cache(str(archive))
 
+    def _is_create_pr_allowed(self, create_pr: bool = False) -> bool:
+        """
+        Check if PR creation is allowed from CLI or from packit.yaml
+        :param create_pr: parameter from CLI --no-pr
+        :return: True if --no-pr in CLI is present or create_pr in packit.yaml is set to True
+                 False
+        """
+        return create_pr if create_pr else self.create_pr
```

Comment:
this also looks like an overkill, we are not saving any lines with this: it can be written inline w/o this method